deepinv/training_utils.py

```diff
@@ -473,10 +490,10 @@ def test(
                         ):
                             if plot_measurements and len(y.shape) == 4:
                                 imgs = [y, x_init, x1, x]
-                                name_imgs = ["Input", "Linear", "Recons.", "GT"]
+                                name_imgs = ["Input", "No learning", "Recons.", "GT"]
```

Comment:
Why not stick to "Linear"? "No learning" looks confusing to me because it may seem like the output of the network at initialization or some method that has frozen parameters

Reply:
because for non-linear operators (e.g. blind deblurring), the reconstruction is not necessarily linear...